README.MD

```diff
@@ -1,18 +1,24 @@
 # Lookup-lc
 
-### Configuration environment:
+To update database `ts-node src/fetch.ts` (debug: `JOB_INTERVAL='* * * * *'` to run every minute)
 
-- `TRUST_KEY` - Trust CRM API key
+To run lookup `ts-node  src/lookup.ts --email=SOME_EMAIL`
+
+Lookup returns:
 
-## Lookup
+- true if email in database
+- false when not found
+- error if anything goes wrong
 
--
-- The request payload should be `{email: "some@email.com"}`
```

Comment:
leave the format of the request, or is this with upstream/trust and not proca-server->lookup?

Reply:
That is for the old lookup (to get status  from LC)